codespell_lib/data/dictionary.txt

```diff
@@ -20009,6 +20041,7 @@ rouine->routine
 rouines->routines
 round-robbin->round-robin
 roundign->rounding
+roundtriped->roundtripped
```

Comment:
Also suggest the space and hyphen separated versions

Reply:
d71d521 + added more variations